style.css

```diff
@@ -0,0 +1,167 @@
+* {
```

Comment:
Olivere samo par sugestija za ubuduce. Generalno treba smanjiti koriscenje fiksiranih visina i sirina u CSS-u na minimum, jer to pravi probleme kod responsive dizajna. Mozes da pogledas Mobile First pristup, a ostavili smo i jos par CSS tutorijala, linkovi su na Trellu. Takodje, mozes stvari kao sto su boje da izvuces u CSS promenljive na pocetku fajla i tako olaksas sebi kasnije. Pogledaj i da li su sve boje ok, jer vidim da u style-guide.md spominju 4 boje a da si ti iskoristio samo 2